bazarr.py

```diff
@@ -116,21 +116,12 @@ def should_stop(self):
 def start_bazarr(process_registry=ProcessRegistry()):
     script = [sys.executable, "-u", os.path.normcase(os.path.join(dir_name, 'bazarr', 'main.py'))] + sys.argv[1:]
 
-    ep = subprocess.Popen(script, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, stdin=subprocess.PIPE)
-    process_registry.register(ep)
     print("Bazarr starting...")
```

Comment:
I've put this to be printed *before* the process is actually started so that logs wouldn't be tangled.